src/ui/main.qml

```diff
@@ -344,6 +344,11 @@ Window {
             Sample[sample].record();
         }
 
+        function onRecordGleanEventWithStringType(sample, type, value) {
+            console.debug("recording Glean event with type", type, "and value", value);
+            Sample[sample].record({[type]: value});
+        }
```

Comment:
I'm wondering if we can make this more generic to handle any number of "extra_keys" e.g. 

```
        Sample.onboardingCtaClick.record({
                                              "panel_id": currentPanelValues._panelId,
                                              "panel_idx": swipeView.currentIndex.toString(),
                                              "panel_cta": ctaObjectName
                                          });
```

And this particular addition happens to just be 1. This would help with my ask for errno and description.

Also, today I learned about computed keys.